include/slurp.h

```diff
@@ -80,7 +81,9 @@ struct slurp_seat {
 	enum wl_pointer_button_state button_state;
 	struct slurp_output *current_output;
 	int32_t x, y;
-	int32_t pressed_x, pressed_y;
+	int32_t anchor_x, anchor_y;
+	int32_t origin_x, origin_y;
+	int32_t origin_anchor_x, origin_anchor_y;
```

Comment:
Hmm, I'm not sure I understand the role of all these variables. What is origin? Do we really need all of these?

Instead, we could compute a delta before updating `state->x` and `state->y`:

```c
int x = /* new x value */;
int dx = state->x - x;
state->x = x;
// If we're editing the anchor, add dx
```

Thoughts?

Reply:
I'm pretty sure we can do it without all those variables but I couldn't find a way before.
I'll try with the delta. Thanks